src/store/auth/authThunk.ts

```diff
@@ -0,0 +1,48 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { AxiosError } from 'axios';
+
+import { instance } from 'services/httpService';
+import { errorHandleService } from 'utils/errorHandleService';
+import { setUserFromLocalStorage } from 'services/localStorage/localStorage';
+import { IRegisterData } from 'interfaces';
+
+export const login = createAsyncThunk(
+  'auth/signin',
+  async (signInData: IRegisterData, { rejectWithValue }) => {
+    const user = {
+      user: {
+        email: signInData.email,
+        password: signInData.password,
+      },
+    };
+    try {
+      const response = await instance.post(`users/login`, user);
+      await setUserFromLocalStorage(response.data);
+      return response.data;
```

Comment:
Try to get all response. Not just the data. Because sometimes you may need "status" property to check your answer